File: fetch.ts
fetch('https://app.devrev.ai/sprint-summariser/vistas/vista-27', {
    method: 'GET',
    headers: {
        'Authorization': 'Bearer ',//enter api token 
        'Content-Type': 'application/json',
    },
})
.then(response => response.json())
.then(data => console.log(data))  // Check the list of sprints/slugs
.catch(error => console.error('Error fetching sprint list:', error));

File: fetch.py
import requests

url = "https://api.devrev.ai/internal/vistas.list"
headers = {
    "Authorization": "Bearer " #enter api token
}
payload = {
    "flavor": ["sprint_board"],
    "limit": 50
}

response = requests.post(url, json=payload, headers=headers)

if response.status_code == 200:
    print("Sprint Boards:")
    print(response.json())
else:
    print("Error:", response.status_code, response.text)
